.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
 node_modules
 dist
-.taskkey
\ No newline at end of file
+.taskkey
+.tool-versions
```

Comment:
I just thought it would be good to add it while updating .gitignore with azure's taskkey, as it's quite common and we have it in other repos too. Can remove if we think it's unnecessary or redundant.